src/cryptenroll/cryptenroll.c

```diff
@@ -358,6 +369,26 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
                 }
 
+                case ARG_TPM2_EXTERNAL_LOCATION:
+                        r = safe_atou32(optarg, &arg_tpm2_external_location);
```

Comment:
are all handles valid? i.e. handle 0 too? should we refuse some invalid handles here?

Reply:
hmm shouldn't this always be parsed in hex? even if unprefixed with `0x`?

Reply:
yes! updated, thanks